dynamodb-table/outputs.tf

```diff
@@ -0,0 +1,11 @@
+output "table_name" {
```

Comment:
If it has 2 regions enabled, should it output where they are? That way running `terraform output` will help with diagnosis?

Reply:
done